internal/handlers.go

```diff
@@ -0,0 +1,81 @@
+package internal
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+func Index(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	fmt.Fprint(w, "Welcome to GoPay!")
```

Comment:
I understand the objective of having this endpoint, but if it is an RESTFUL API I would return just json to keep the consistent of the API